packages/buidler-etherscan/src/ABITypes.ts

```diff
@@ -0,0 +1,48 @@
+export interface ABIArgumentLengthError extends Error {
```

Comment:
I find this file very confusing. The interfaces defined here aren't actually used, but their type guards are.

Reply:
They are used while handling the error and describing it in an error message. These custom messages might not be necessary since the errors are now set as parents for a buidler plugin error.